x/auth/client/cli/tx_multisign.go

```diff
@@ -148,20 +140,12 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 						Value:   anyPk.Value,
 					},
 				}
-				txBytes, err := txCfg.TxEncoder()(txBuilder.GetTx())
-				if err != nil {
-					return err
-				}
-				decodedTx, err := decoder.Decode(txBytes)
-				if err != nil {
-					return err
-				}
-				txData := txsigning.TxData{
-					Body:          decodedTx.Tx.Body,
-					AuthInfo:      decodedTx.Tx.AuthInfo,
-					AuthInfoBytes: decodedTx.TxRaw.AuthInfoBytes,
-					BodyBytes:     decodedTx.TxRaw.BodyBytes,
+				builtTx := txBuilder.GetTx()
+				adaptableTx, ok := builtTx.(signing.V2AdaptableTx)
+				if !ok {
+					return fmt.Errorf("expected Tx to be a V2AdaptableTx")
```

Comment:
Debugging this error is slightly easier when including the type of builtTx, like you did later on L333.